sdk/marketplace/Azure.ResourceManager.Marketplace/src/autorest.md

```diff
@@ -16,7 +16,62 @@ skip-csproj: true
 modelerfour:
   flatten-payloads: false
 
- 
+rename-mapping:
+  AcknowledgeOfferNotificationProperties: AcknowledgeOfferNotificationContent
+  AcknowledgeOfferNotificationProperties.properties.acknowledge: IsAcknowledgeActionFlagEnabled
+  AcknowledgeOfferNotificationProperties.properties.dismiss: IsDismissActionFlagEnabled
+  AcknowledgeOfferNotificationProperties.properties.removeOffer: IsRemoveOfferActionFlagEnabled
+  Accessibility: PrivateStorePlanAccessibility
+  AdminAction: MarketplaceAdminAction
+  AdminRequestApprovalsResource: MarketplaceAdminApprovalRequest
```

Comment:
I saw a lot of resources and models are renamed and their main change is to interchange the order of `Approval` and `Request`. Why do we need to do that?

Reply:
It's bad name in swagger spec, according to description and actual operations, all these a `Request`, so made this rename changes